src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/YFlowAllEntityPaths.groovy

```diff
@@ -0,0 +1,64 @@
+package org.openkilda.functionaltests.helpers.model
+
+import org.openkilda.model.SwitchId
+import org.openkilda.northbound.dto.v2.yflows.YFlowPaths
+import org.openkilda.testing.model.topology.TopologyDefinition
+
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import groovy.transform.builder.Builder
+
+@EqualsAndHashCode(excludes = "topologyDefinition")
+@Builder
+@ToString(includeNames = true, excludes = 'topologyDefinition', includePackage = false)
+class YFlowAllEntityPaths {
```

Comment:
I am thinking of reusing HaFlowAllEntityPaths and renaming it to ComplexFlowAllEntityPath and having RegularFlowAllEntityPath for the regular flow. WDYT?